insonmnia/worker/overseer.go

```diff
@@ -657,3 +663,109 @@ func (o *overseer) OnDealFinish(ctx context.Context, containerID string) error {
 func (o *overseer) Logs(ctx context.Context, id string, opts types.ContainerLogsOptions) (io.ReadCloser, error) {
 	return o.client.ContainerLogs(ctx, id, opts)
 }
+
+func (o *overseer) removeImageRepoAndTag(image io.Reader) (io.Reader, error) {
```

Comment:
This function should be out of the `overseer` scope because it doesn't use any of its fields nor methods.